reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -220,56 +234,62 @@ Servers.txt file and then sets the startup type of the WinRM service on all
 of the computers to Automatic.
 
 ```powershell
-C:\PS> $servers = Get-Content servers.txt
-C:\PS> Set-Service WinRM -ComputerName $servers -startuptype Automatic
+$servers = Get-Content servers.txt
+Set-Service WinRM -ComputerName $servers -startuptype Automatic
 ```
 
-To see the results use the Get-WMIObject cmdlet with the Win32_Service object.
-For more information, see Set-Service.
+To see the results use the `Get-WMIObject` cmdlet with the **Win32_Service**
+object. For more information, see
+[Set-Service](../../Microsoft.PowerShell.Management/Set-Service.md).
 
 ### HOW TO RECREATE THE DEFAULT SESSION CONFIGURATIONS
 
+```
 ERROR:  ACCESS IS DENIED
+```
 
 To connect to the local computer and run commands remotely, the local
 computer must include session configurations for remote commands.
 
-When you use Enable-PSRemoting, it creates default session configurations
+When you use `Enable-PSRemoting`, it creates default session configurations
 on the local computer. Remote users use these session configurations
 whenever a remote command does not include the ConfigurationName parameter.
 
 If the default configurations on a computer are unregistered or deleted,
-use the Enable-PSRemoting cmdlet to recreate them. You can use this cmdlet
+use the `Enable-PSRemoting` cmdlet to recreate them. You can use this cmdlet
 repeatedly. It does not generate errors if a feature is already configured.
 
 If you change the default session configurations and want to restore the
 original default session configurations, use the
-Unregister-PSSessionConfiguration cmdlet to delete the changed session
-configurations and then use the Enable-PSRemoting cmdlet to restore them.
-Enable-PSRemoting does not change existing session configurations.
+`Unregister-PSSessionConfiguration` cmdlet to delete the changed session
+configurations and then use the `Enable-PSRemoting` cmdlet to restore them.
+`Enable-PSRemoting` does not change existing session configurations.
 
-Note: When Enable-PSRemoting restores the default session configuration, it
-does not create explicit security descriptors for the configurations.
-Instead, the configurations inherit the security descriptor of the RootSDDL,
-which is secure by default.
+> [!NOTE]
+> When `Enable-PSRemoting` restores the default session configuration, it
+> does not create explicit security descriptors for the configurations.
+> Instead, the configurations inherit the security descriptor of the
+> RootSDDL, which is secure by default.
 
 To see the RootSDDL security descriptor, type:
 
 ```powershell
 Get-Item wsman:\localhost\Service\RootSDDL
 ```
 
-To change the RootSDDL, use the Set-Item cmdlet in the WSMan: drive. To
+To change the RootSDDL, use the `Set-Item` cmdlet in the WSMan: drive. To
 change the security descriptor of a session configuration, use the
-Set-PSSessionConfiguration cmdlet with the SecurityDescriptorSDDL or
+`Set-PSSessionConfiguration` cmdlet with the SecurityDescriptorSDDL or
```

Comment:
If SecurityDescriptorSDDL is a parameter of the cmdlet, should be in bold.

Reply:
Fixed